include/aspect/utilities.h

```diff
@@ -813,6 +813,18 @@ namespace aspect
          */
         std_cxx11::unique_ptr<aspect::Utilities::AsciiDataLookup<1> > lookup;
     };
+
+    /**
+     * This function computes a factor which can be used to make sure that the
+     * Jacobian remains positive definite. The factor is defined by
+     * $\frac{2\eta(\varepsilon(\bm u))}{\left[1-\frac{b:a}{\|a\| \|b\|} \right]^2\|a\|\|b\|}$,
+     * where $a=\varepsilon(\bm u)$ and $b=\frac{\partial\eta(\varepsilon(\bm u),p)}.
```

Comment:
the denominator of `b` is missing here.

I think what I would like to say here is something like this after the first sentence:
```
  The goal of this function is to find a factor $\alpha$ so that
  $2\eta I + \alpha ****$ remains a positive definite matrix. Here, $a=...$ is the @p strain_rate
  and $b=....$ is the derivative of the viscosity with respect to the strain rate and is given by
  @p dviscosities_dstrain_rate. Since the viscosity $\eta$ must be positive, there is always a
  value of $\alpha$ (possibly small) so that the result is a positive definite matrix. In the best
  case, we want to choose $\alpha=1$ because that corresponds to the full Newton step,
  and so the function never returns anything larger than one.
```
(where you want to complete the formulas).